src/task.c

```diff
@@ -294,6 +294,9 @@ void JL_NORETURN jl_finish_task(jl_task_t *t)
 {
     jl_task_t *ct = jl_current_task;
     JL_PROBE_RT_FINISH_TASK(ct);
+    uint64_t now = jl_hrtime();
+    ct->cpu_time_ns += now - ct->last_scheduled_ns;
+    ct->last_scheduled_ns = now;
```

Comment:
Should it really set this field when the task is finished? It's more like when it was descheduled.. but I guess it could still be cool to know when it finished, so probably can't hurt to abuse the field for this

Reply:
yeah I don't think that was necessary